README.md

```diff
@@ -83,7 +83,8 @@ branches and bookmarks. Mercurial branches are permanent markers on each
 changeset that belongs to them, and bookmarks are similar to git branches.
 
 You may choose how to interact with those with the `cinnabar.refs`
-configuration. The following values are supported:
+configuration. The following values are supported, either individually or in
+combination:
```

Comment:
(comma-separated)

Reply:
Good catch, I meant to include that before